### PR TITLE
New workflow cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips

### PR DESCRIPTION
**Pull request by wangke19**
Add new workflow cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips, in workflow list of cluster-bot, there is only one fips related workflow.

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - chain: ipi-conf-gcp
+  - ref: ovn-conf 
+  - ref: ovn-conf-ipsec-manifest
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  env:
+  - name: FIPS_ENABLED
+    default: "true"
+  documentation: |-
+    Create an fips enabled IPI cluster with OVN and ipsec for QE e2e tests.
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml:13]: For fips, there is no need to add a new chain/workflow now.
You can add env `FIPS_ENABLED: "true" in the config directly for an existing chain/workflow.
```
- as: aws-c2s-ipi-disconnected-private-fips-p2-f14
  cron: 32 14 11,26 * *
  remote_api: true
  steps:
    cluster_profile: aws-c2s-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
      FIPS_ENABLED: "true"
    test:
    - chain: cucushift-installer-check-cluster-health
    workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
```

Comment (wangke19) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml:13]: yes, I just add one more fips workflow for cluster-bot command line workflow-launch. Currently, we just have one fips workflow available.

Comment (wangke19) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml:13]: Detail see, https://github.com/openshift/release/blob/master/core-services/ci-chat-bot/workflows-config.yaml

Comment (liangxia) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml:13]: Workflow does not need to specify fips in it now. E.g, for a default non-fips workflow, fips can be enabled by adding env `FIPS_ENABLED=true`

Comment (wangke19) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml:13]: That's okay. Closing this PR.

**Comment by wangke19** [comment]
We can add parameter FIPS_ENABLED=true to workflow-launch, no need one separate workflow. Closing the PR.

**Pull request closed without merging**